### PR TITLE
Re-Add/Expand on FourJR Plugins

### PR DESCRIPTION
**Pull request by Coolguy3289**

### CODE REVIEW

**Comment by Coolguy3289** [comment]
Awaiting merge of PR on FourJR's branch before marking ready for review.

File: plugins/registry.json [hunk 1]
```diff
@@ -25,5 +25,79 @@
...
+        "thumbnail_url": "https://i.imgur.com/Mo60CdK.png"
+    },
+     "claim": {
+        "repository": "fourjr/modmail-plugins",
+        "branch": "master",
+        "description": "Allows supporters to claim thread by sending ?claim in the thread channel",
+        "title": "Claim Thread",
+        "icon_url": "https://i.imgur.com/Mo60CdK.png",
+        "thumbnail_url": "https://i.imgur.com/Mo60CdK.png"
+    },
```
Comment (Jerrie-Aries) [inline plugins/registry.json:70]:
```suggestion
    },
```

Duplicate :)

**Review by Jerrie-Aries — CHANGES_REQUESTED** [review]

**Review by fourjr — CHANGES_REQUESTED** [review]
`bot_version` has to be set to `4.0.0` and branch should be `v4`.

**Comment by Coolguy3289** [comment]
Requested changes implemented.

**Review by fourjr — APPROVED** [review]

**Pull request merged**